htdocs/luci-static/resources/homeproxy.js

```diff
@@ -176,6 +176,24 @@ return baseclass.extend({
 		);
 	},
 
+	generateRand: function(type, length) {
+		['base64', 'hex'].includes(type)
+			var byteArr = crypto.getRandomValues(new Uint8Array(length));
```

Comment:
```suggestion
		var byteArr;
		if (['base64', 'hex'].includes(type))
			byteArr = crypto.getRandomValues(new Uint8Array(length));
```
make it more readable